apps/coscrad-frontend-e2e/src/support/commands.ts

```diff
@@ -6,13 +8,59 @@ declare namespace Cypress {
     interface Chainable<Subject> {
         login(email: string, password: string): void;
     }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    interface Chainable<Subject> {
+        grantPermissions(permissions: CoscradBrowserPermissions): void;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Still a bit foggy on chaining, out of curiosity, why is it necessary to redeclare this Chainable interface rather than having grantPermissions, assertValueCopiedToClipboard, and getByDataAttribute all within one Chainable interface statement?